tools/sgcollect_info

```diff
@@ -38,6 +38,14 @@ from tasks import add_file_task
 
 import password_remover
 
+import ssl
+try:
+    _create_unverified_https_context = ssl._create_unverified_context
+except AttributeError:
+    pass
```

Comment:
Can this log a warning like "Unable to allow SSL self-signed certs"?

Reply:
This gets thrown if running an older version (as a fallback) which won't validate certs. The collection will succeed in this case.